templates/app/node14-adonis.yml

```diff
@@ -1,6 +1,6 @@
 services:
   app:
-    image: kooldev/node:14-adonis
+    image: kooldev/node:16-adonis
```

Comment:
we should rename this file to `node-adonis.yml` and change its references as well